src/pages/Accounts/components/AccountEntryRow.tsx

```diff
@@ -59,10 +61,10 @@ export const AccountEntryRow: React.FC<AccountEntryRowProps> = ({
 
   const assetIdOrIconSrcProps = useMemo(
     () =>
-      asset?.icons
-        ? { assetId, name: title, pairProps: { showFirst: true } }
-        : { src: icon, name: title, bg: asset?.color },
-    [asset?.icons, assetId, icon, title, asset?.color],
+      assetId
```

Comment:
Needed because an assets without multiple icons will use the FOX icon, I think we didn't have this usage yet... It's effectively a bugfix

Reply:
Confirmed we're still looking good here!

<img width="1644" alt="image" src="https://github.com/user-attachments/assets/3dab8607-bdf8-478a-b1d1-491af96f3d3d">